doc/overview.rst

```diff
@@ -73,6 +73,90 @@ produced.
 Physics
 =======
 
+Celeritas implements physics processes and models for transporting electron, positron,
+and gamma particles as shown in the accompanying table. Implementation details of these models 
+and their corresponding Geant4 classes are documented in :ref:`celeritas_physics`.
+
+.. only:: html
+
+   .. table:: Electromagnetic physics processes and models available in Celeritas.
+
+      +----------------+---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      | **Particle**   | **Processes**       | **Models**                | **Celeritas Implementation**                  | **Applicability**        |
+      +----------------+---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |:math:`e^-`     | Ionisation          | Moller                    | :cpp:class:`celeritas::MollerBhabhaModel`     |       0 - 100 TeV        |
```

Comment:
Since we're not including Breathe documentation for the Models, these links don't work. We have three options I think:
- Change the links point to the Interactor
- Move the documentation from Interactor to Model and replace the Interactor links with Model
- Forget the links and just drop `:cpp:class:celeritas::`

Reply:
Given the Interactor holds the actual implementation, maybe point to these instead of the models (which are more frontends?)?

Reply:
Sounds good to me. If we ever generate our own cross sections, the "model" implementation may be split off into a couple of classes; but for now at least the Interactor is where the magic happens. I think in the case of MSC we should link to UrbanMscScatter.

Reply:
O.k., done! BTW, thanks for the LaTeX build fix, could not work out why Sphinx was introducing a line break at that point in `GammaDistribution.hh`!

Reply:
Apparently line breaks inside of `<em>` doxygen blocks don't get ignored inside Doxygen and end up propagating through as paragraphs... yuck.